### PR TITLE
chore: update README, CHANGELOG, appcasts

### PR DESCRIPTION
**Pull request by lotem**
WIP: commit this after adding new color schemes.

### CODE REVIEW

File: update/appcast.xml [hunk 1]
```diff
@@ -1,15 +1,15 @@
...
+    <link>http://rime.github.io/release/weasel/appcast.xml</link>
+    <description>小狼毫 Appcast 更新頻道</description>
     <language>zh</language>
     <item>
       <title>小狼毫 0.9.30</title>
-      <sparkle:releaseNotesLink>http://rimeime.github.io/weasel-update/</sparkle:releaseNotesLink>
+      <sparkle:releaseNotesLink>http://rime.github.io/release/weasel/</sparkle:releaseNotesLink>
       <pubDate>Tue, 1 Apr 2014 00:00:00 +0800</pubDate>
-      <enclosure url="http://dl.bintray.com/lotem/rime/weasel-0.9.30-installer.exe?direct"
+      <enclosure url="http://dl.bintray.com/lotem/rime/weasel-0.9.30.0-installer.exe?direct"
```
Comment (Prcuvu) [inline update/appcast.xml]: Please consider moving installer package under the name of organization.
Reference: [Manage your Bintray and GitHub organizations better together | Blog @Bintray](https://blog.bintray.com/2015/11/05/manage-your-bintray-and-github-organizations-better-together/)

Comment (lotem) [reply update/appcast.xml]: Acknowledged.
Created an empty organization. Let's use it for the next version. Do you have an account? I need to invite people individually by account.

Comment (Prcuvu) [reply update/appcast.xml]: I have just registered a Bintray account named "prcuvu".

File: CHANGELOG.md [hunk 2]
```diff
@@ -8,32 +8,34 @@
...
 * **tsf:**
-  *  fix candidate selecting in preview preedit mode ([206efd69](https://github.com/rime/weasel/commit/206efd692124339d0e256198360c1860c72cd807))
-  *  support user defined preedit display type ([f76379b0](https://github.com/rime/weasel/commit/f76379b01abe9d3971d68e2e272067e0bb855cc9))
+  *  fix candidate selecting in preview preedit mode ([206efd69](206efd69))
+  *  support user defined preedit display type ([f76379b0](f76379b0))
+
+#### Bug Fixes
+
+*   inline_preedit && fullscreen causing dead lock when there's no candidates. ([deb0bb24](deb0bb24))
+* **RimeWithWeasel:**  fix wild pointer ([ae2e3c4a](ae2e3c4a))
```
Comment (nameoverflow) [inline CHANGELOG.md]: Bugs introduced after the last release might not be contained?

Comment (Prcuvu) [reply CHANGELOG.md]: Bugs introduced after 0.9.30 release and fixed before 0.10.0 are not visible to users, therefore should not be mentioned in changelog.

Comment (lotem) [reply CHANGELOG.md]: Make sense. But how do user report those bugs?
As we hold the assumption that all users are upgrading from previous releases, our workflow should make that assumption true.

Comment (lotem) [reply CHANGELOG.md]: Removed. Please take another look.

**Comment by nameoverflow** [comment]
Well, some fixes were not counted in conventional-changelog, such as https://github.com/rime/weasel/issues/28 / https://github.com/rime/weasel/issues/107

**Comment by lotem** [comment]
Manually added two closed bugs.
The change log generator is based on git commit history. It doesn't read GitHub issues.

**Pull request merged**